crates/astria-sequencer/src/config.rs

```diff
@@ -5,15 +5,20 @@ use serde::{
 };
 
 const DEFAULT_ABCI_LISTEN_ADDR: &str = "127.0.0.1:26658";
+const DEFAULT_GENESIS_ACCOUNT_ALLOCATIONS_PATH: &str = "accounts-genesis.csv";
+const DEFAULT_COMETBFT_GENESIS_FILE: &str = "/.cometbft/config/genesis.json";
 
 #[derive(Debug, Deserialize, Parser, Serialize)]
 pub struct Config {
     /// The endpoint on which Sequencer will listen for ABCI requests
     #[arg(long, default_value_t = String::from(DEFAULT_ABCI_LISTEN_ADDR))]
     pub(crate) listen_addr: String,
     /// The path to the json encoded genesis file with a list of accounts.
```

Comment:
I think this comment is not correct given that this now assumes a csv. However, I would probably just leave it as a json and not introduce extra dependencies.